lte/gateway/release/build-magma.sh

```diff
@@ -131,6 +131,9 @@ OAI_DEPS=(
 # OVS runtime dependencies
 OVS_DEPS=(
     "magma-libfluid >= 0.1.0.4"
+    "libopenvswitch >= 2.8.9"
+    "openvswitch-switch >= 2.8.9"
+    "openvswitch-datapath-source >= 2.8.9"
```

Comment:
we need openvswitch-datapath-module-4.9.0-9-amd64_2.8.9-1_amd64.deb package to get OVS fix. Can we add dependency on that ?